content/project/impl_1/rcp.py

```diff
@@ -0,0 +1,26 @@
+from socketserver import BaseRequestHandler, TCPServer
+from jsonschema import validate
+import time
+import json
+
+
+class EchoHandler(BaseRequestHandler):
+    def handle(self):
+        print('Got connection from', self.client_address)
+        while True:
+            msg = self.request.recv(8192)
```

Comment:
@postprandial Let's add a comment to this line saying: "8192 refers to the number of received bytes to get from the request in one data block"

Reply:
Good idea!

Reply:
Added comment @abigailcx